src/atoms/story-elements/text/text.tsx

```diff
@@ -44,6 +44,48 @@ export const StyledText = styled.div<StoryElementProps>`
   }
 `;
 
+export const StyledText = styled.div<StoryElementProps>`
```

Comment:
Instead of having a separate styled component for visual story can pass a prop to the styled component like [this](https://stackoverflow.com/questions/52321539/react-passing-props-with-styled-components) 
There will be lesser repetation